java/ql/src/semmle/code/java/frameworks/apache/Exec.qll

```diff
@@ -6,16 +6,20 @@ library class TypeCommandLine extends Class {
   TypeCommandLine() { hasQualifiedName("org.apache.commons.exec", "CommandLine") }
 }
 
-library class MethodCommandLineParse extends ExecMethod {
+library class MethodCommandLineParse extends Method, ExecCallable {
```

Comment:
Let's get rid of the archaic `library` annotations while we're editing this file. And a few qldoc additions seems like easy inclusions as well.

Reply:
Shall I replace `library` with `private`?

Reply:
If the class isn't referenced anywhere else (i.e. things still compile), then yes, I think that's fine (even though it technically is a breaking change).